modules/light-clients/08-wasm/types/wasm_test.go

```diff
@@ -41,7 +42,8 @@ func (suite *TypesTestSuite) TestGetCodeHashes() {
 
 			tc.malleate()
 
-			codeHashes := types.GetCodeHashes(suite.chainA.GetContext(), GetSimApp(suite.chainA).AppCodec())
```

Comment:
sorry, i know this wasn't in your PR but also, do we need `tc := tc` above on line 38?

Reply:
Not sure, I'm in favour of leaving it as is. Doesn't hurt anything